### ISSUE TITLE
pause ads

### ISSUE THREAD

**Issue by d7coders** [zhw2590582/ArtPlayer#546]
is it possible to pause ads?

**Comment by zhw2590582** [comment]
`art.plugins.artplayerPluginAds.pause()`

By the way, `artplayerPluginAds` is currently being refactored

**Comment by d7coders** [comment]
I got this error, I'm using vueJs 3

Uncaught TypeError: 'get' on proxy: property 'artplayerPluginAds' is a read-only and non-configurable data property on the proxy target but the proxy did not return its actual value (expected '#<Object>' but got '#<Object>')

**Comment by zhw2590582** [comment]
Yes, there seems to be something bugs with this plugin, I need to refactor it
